src/graphql/data/util.tsx

```diff
@@ -120,7 +120,7 @@ export function validateUrlChainParam(chainName: string | undefined) {
 }
 
 // TODO(cartcrom): refactor into safer lookup & replace usage
-export const CHAIN_NAME_TO_CHAIN_ID: { [key in Chain]: SupportedChainId } = {
+export const CHAIN_NAME_TO_CHAIN_ID: { [key in Chain]?: SupportedChainId } = {
```

Comment:
with the addition of `?` the keys don't have to be exhaustive which makes this future proof for more chain additions. the cost is that the return type is now optional